File: DailyFlash Solutions/DailyFlash Solutions/06-DailyFlash_Solutions/20_Jan_Solutions_One/Cpp/oddeven.cpp
/*
Program 3 : Write a Program that accepts a n inte ger from user and prints
whether that number is odd or even .
Input: 2
Output: 2 is an Even Number

*/


#include<iostream>

int main(){
	
	int num;

	printf("Enter the number ");
	scanf("%d",&num);

	if(num%2==0){
		printf("%d is Even Number",num);
	}else{
		printf("%d is Odd Number",num);
	}	
}	

File: DailyFlash Solutions/DailyFlash Solutions/06-DailyFlash_Solutions/20_Jan_Solutions_One/JAVA/OddEven.java
/*
Program 3 : Write a Program that accepts a n inte ger from user and prints
whether that number is odd or even .
Input: 2
Output: 2 is an Even Number

*/

import java.io.*;

class OddEven{
	
	public static void main(String args[])throws IOException{

		int num;
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Enter the number");
		num = Integer.parseInt(br.readLine());


		if(num%2==0){
			System.out.printf("%d is Even Number",num);
		}else{
			System.out.printf("%d is Odd Number",num);
		}
		
	}	
}	
